src/app/public/modules/file-attachment/file-attachment.component.ts

```diff
@@ -286,14 +281,6 @@ export class SkyFileAttachmentComponent implements ControlValueAccessor, AfterVi
     this.changeDetector.markForCheck();
   }
 
-  public validate(control: AbstractControl): { [key: string]: any } {
-    if (!this.control) {
-      this.control = control;
-    }
-
-    return undefined;
-  }
-
```

Comment:
☝️ Our validator wasn't actually doing any validation - it was just used to get the `AbstractControl`. Now that we are using the `NgControl` injection pattern, this is no longer needed.